src/EOFeedVerifier.sol

```diff
@@ -30,17 +28,16 @@ using Merkle for bytes32;
  * sufficient voting power.
  */
 contract EOFeedVerifier is IEOFeedVerifier, OwnableUpgradeable {
-    bytes32 public constant DOMAIN = keccak256("DOMAIN_CHECKPOINT_MANAGER");
-    uint256 public constant MIN_VALIDATORS = 3;
+    bytes32 public constant DOMAIN = keccak256("EORACLE_FEED_VERIFIER");
+
+    uint256 internal _minNumOfValidators;
```

Comment:
why constant is changed to storage variable? though we even don't have any setter for it..

Reply:
@ArielLaub do we need this change? I dont see why